Replaced bootstrap-tour with introJs tour library in Story Activity

abhishek-sultaniya opened the pull request:
Fixes: #1090 
I have replaced bootstrap-tour with introJs tour library in Story Activity.All the suggestions,improvements and modifications are welcome.I followed all the steps as described in [here](https://github.com/llaske/sugarizer/pull/1083#issuecomment-1167648209).

disha1202 commented on activities/Story.activity/js/components/SugarTutorial.js:
```diff
@@ -1,62 +1,35 @@
+      l10n: {
+        stringTutoPrev: "Prev",
+        stringTutoNext: "Next",
+        stringTutoEnd: "End",
```
stringTutoEnd can be removed as we're no longer using it.

disha1202 commented on activities/Story.activity/js/components/SugarTutorial.js:
```diff
@@ -1,62 +1,35 @@
+  },
+  methods: {
+    show: function (steps) {
+      introJs()
```
Filter the steps before starting the tutorial.
https://github.com/llaske/sugarizer/pull/1083/files#diff-43815ec1886368b47450dfe256cc92d8594fd7c643a362644fc9aeeef5fbc519R22

disha1202 reviewed (changes requested):
This is great work 👏🏻 I have suggested some minor changes which you can incorporate.

abhishek-sultaniya commented:
Thanks, @disha1202 , Now corrected these two suggestions in the recent commit:

Removed stringTutoEnd SugarTutorial.js
Now the steps are filtered before starting the tutorial.

llaske commented:
@abhishek-sultaniya I think some styles are missing. The size of the header is too small and I think the text spacing is not the same.

![image](https://user-images.githubusercontent.com/981841/188285661-1196d985-7a8c-48b9-9ce8-6d21798191f1.png)

abhishek-sultaniya commented:
Hey @llaske ,In recent commit I have changed the style.After the commit it looks as follows:-
![image](https://user-images.githubusercontent.com/80109306/188295101-44543d85-3520-48ba-8468-a0df76eaabd6.png)

llaske commented:
It's better but I wonder about the close button position like I've mentioned [here](https://github.com/llaske/sugarizer/pull/1099#issuecomment-1236190370).
Plus why it will be need to specify `line-height` in this activity and not elsewhere?

disha1202 commented:
> It's better but I wonder about the close button position like I've mentioned [here](https://github.com/llaske/sugarizer/pull/1099#issuecomment-1236190370).
 Plus why it will be need to specify `line-height` in this activity and not elsewhere?
 
 I faced the same issue with header, this is also because of different versions of IntroJs
 https://github.com/usablica/intro.js/pull/1708/files#diff-61919747715934800867ffcb198d7e8a6450cb3fbd4c8902cd3afbbc807a6d60R203

disha1202 commented:
> It's better but I wonder about the close button position like I've mentioned [here](https://github.com/llaske/sugarizer/pull/1099#issuecomment-1236190370).
 Plus why it will be need to specify `line-height` in this activity and not elsewhere?
 
 I faced the same issue with header, this is also because of different version of IntroJs
 https://github.com/usablica/intro.js/pull/1708/files#diff-61919747715934800867ffcb198d7e8a6450cb3fbd4c8902cd3afbbc807a6d60R203

abhishek-sultaniya commented:
I think it is due to different versions of introjs.What should I do,handle it using custom css or try some previous version of introjs?

llaske commented:
Try to use the same version than the one currently use in Sugarizer to see if it's solve the issue.

abhishek-sultaniya commented:
> Try to use the same version than the one currently use in Sugarizer to see if it's solve the issue.

Hello @llaske ,in recent commits I have used the version of introjs used in SUgarizer and removed the line-height from activity.css.
Please review it.

llaske commented:
It works. Thanks.
